src/Equinor.ProCoSys.IPO.Query/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQuery.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Equinor.ProCoSys.IPO.Domain;
+using MediatR;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.GetOutstandingIpos
+{
+    public class GetOutstandingIposForCurrentPersonQuery : IRequest<Result<OutstandingIposResultDto>>, IProjectRequest
```

Comment:
Add tests for AccessValidator

Reply:
Not valid anymore since we're no longer taking in projectName